### PR TITLE
creating a file for krishna meduri

### LINKED ISSUE #13: Add a hometown file for kmeduri

### PR DESCRIPTION
**Pull request by kmeduri**
Resolved #13 
- [x] Create a branch
- [x] Added a file
- [x] Committed the file
- [x] Opened a Pull Request
- [x] Have a discussion
- [x] Make more commits
- [x] Make a pull request

### CODE REVIEW

File: places/kmeduri-hometown.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Cupertino CA
+
```
Comment (kmeduri) [inline places/kmeduri-hometown.md:2]: i love the counter

File: places/kmeduri-hometown.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+## Great places to eat
```
Comment (githubteacher) [inline places/kmeduri-hometown.md:3]: @kmeduri I :heart: sushi - any good recommendations :sushi:

Comment (kmeduri) [inline places/kmeduri-hometown.md:3]: @githubteacher I have no good recommendations for sushi

**Comment by githubteacher** [comment]
@kmeduri this looks :sparkles: for a 1st draft! :shipit:

**Pull request merged**